src/main/java/org/isf/patient/model/Patient.java

```diff
@@ -552,13 +550,17 @@ public String getSearchString() {
 		sbName.append(' ');
 		sbName.append(getCity().toLowerCase());
 		sbName.append(' ');
-		if (getAddress() != null) sbName.append(getAddress().toLowerCase()).append(' ');
-		if (getTelephone() != null) sbName.append(getTelephone()).append(' ');
-		if (getNote() != null) sbName.append(getNote().toLowerCase()).append(' ');
-		if (getTaxCode() != null) sbName.append(getTaxCode().toLowerCase()).append(' ');
+		if (getAddress() != null)
```

Comment:
If this is going to be reformatted then please add the project standard of using opening and closing braces for `if` statements; thanks